.werft/vm/vm.ts

```diff
@@ -114,8 +114,7 @@ export function startVM(options: { name: string }) {
  * @returns true if the VM already exists
  */
 export function vmExists(options: { name: string }) {
```

Comment:
I would expect that `name` is the `destname` which will be without the leading `preview-`. What am I missing here?

Reply:
Is it worth digging deeper?